_posts/blog/2019-03-14/2019-03-14-how-to-sell-digital-goods-for-cryptocurrency-on-openbazaar.markdown

```diff
@@ -0,0 +1,69 @@
+
+Give it a great **title** and **descriptio**n that relates to keywords people may use in search to find it easily. Make sure to also describe how you will send the file to users when they complete their purchase so they know what to expect and where, whether in OpenBazaar chat or via email. It’s also helpful to include any instructions they may need for setting up or installing your file if it requires that.
+
+Include at least 1 great **image** that describes your item such as a book jacket mockup for an ebook or the cover art for a music album. 
+
+Leave the **shipping** option blank because you don’t need to worry about shipping! Hooray!
+
+**Tags** can be added to boost the searchability of your item. Include obvious ones that relate to the item itself like “book” or “music” then more as relates to broader categories it represents such as “fiction” or “bluegrass.”
+
+A **category** is to help keep your store organized. If you plan to sell both ebooks AND music albums, you’d just type in the corresponding category for each of your listings.
+
+**Variants** and **inventory** can likely be left blank as well. These are helpful for things like physical goods where you may offer the same hat but in 3 different colors with only a certian number of each available. If you have variations with your digital goods it may be worth creating separate listings for each.
+
+With the **return policy** you can set this up however you’d like. You may want to say no refunds if the price point is very low on the item or perhaps offer a 100% satisfaction guarantee with a full refund option. 
+
+**Terms & conditions** can be added if you have any conditions or advisory you want to make people aware of when accessing or using your digital good.
+
+Select the **coins** you’d like to receive as payment--it can be just one of them or any combination.
+
+#### 3. Share your item from OpenBazaar.com 
+
+OpenBazaar is a marketplace where people can browse and discover the items that you have for sale—but that’s not the only way they can find them! [OpenBazaar.com](https://openbazaar.com) exists to make it easy for you and others to share stores and listings to your social media and other communities.
+
+Once your listing is live you can click "View on web" in the address bar at the top to see it and share it.
+
+##### More Than a Marketplace
+
+The software is still new and it is very niche because it exclusively uses cryptocurrency for payments. It needs every true believer’s help to grow into a phenomenon worldwide and that starts with our community. Make sure to connect with us to market your store and connect with like-minded people all aroudn the world. Meet us on [Twitter](https://twitter.com/openbazaar), [Reddit](https://reddit.com/r/openbazaar), [Slack](https://openbazaar.org/slack) and [Telegram](https://t.me/OpenBazaarGroup)!
```

Comment:
"connect"is used twice in the same sentence, it would be good to use a synonym for one of them.

"aroudn" should be "around"